migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -207,7 +207,7 @@ For more information, see xref:../observability/logging/cluster-logging.adoc#clu
 
 Some logging configurations that were available in {product-title} 3.11 are no longer supported in {product-title} {product-version}.
 
-For more information on the explicitly unsupported logging cases, see the xref:../observability/logging/cluster-logging-support.adoc#cluster-logging-support[logging support documentation].
+For more information on the explicitly unsupported logging cases, see the xref:../observability/logging/logging_release_notes/cluster-logging-support.adoc#cluster-logging-support[logging support documentation].
```

Comment:
I realize this is an update to the URL but isn't this kind of a double negative.  Do we really identify the "unsupported" cases which could be significantly larger the the "supported" cases?  IMO we are explicit about what we support and everything else is inferred to be "unsupported"

Reply:
This part of documentation is not maintained by logging docs. The reason I needed to update the reference here is because i moved the referenced file to another directory and xrefs are based on file path.